### ISSUE TITLE
unplugin-element-plus不适配vite4.0.0

### ISSUE THREAD

**Issue by XAllure** [element-plus/unplugin-element-plus#174]
└─┬ unplugin-element-plus 0.4.1
  └─┬ unplugin 0.7.2
    └── ✕ unmet peer vite@"^2.3.0 || ^3.0.0-0": found 4.0.1

好像不适配vite4.0.0

**Comment by mPyth** [comment]
I've got similar problem in a try to update my Vue + ElementPlus project. 
I had intention to update Vue.js from 3.2.37 to 3.2.45, ElementPlus from 2.2.15 to 2.2.27, and UnpluginElementPlus is not changed (0.4.1).
I've tried to create new Vue project from scratch (by calling `npm init vue@latest`) and to copy old code to the new project.
After all this steps "npm run dev" does NOT work, and inside `vite.config.ts` on line where ElementPlus is added to array of plugins next error is reported from VSCode:
```
Type 'Plugin_2' is not assignable to type 'PluginOption'.
  Type 'import(".../projectFolder/node_modules/unplugin-element-plus/node_modules/vite/dist/node/index").Plugin' is not assignable to type 'import(".../projectFolder/node_modules/vite/dist/node/index").Plugin'.ts(2322)
```
In the process of update Vite is update from 3.0.4 to 4.0.0. The main change in Vite is moving from Rollup 2 to Rollup 3.
Potential problem could be Rollup update and Plugin API changes, more details [here](https://rollupjs.org/guide/en/#changes-to-the-plugin-api) .
